ibm/service/vpc/resource_ibm_is_bare_metal_server.go

```diff
@@ -244,9 +244,11 @@ func ResourceIBMIsBareMetalServer() *schema.Resource {
 							Description: "If true, the VPC infrastructure performs any needed NAT operations. If false, the packet is passed unmodified to/from the network interface, allowing the workload to perform any needed NAT operations.",
 						},
 						isBareMetalServerNicInterfaceType: {
-							Type:        schema.TypeString,
```

Comment:
Can you please implement the acceptance tests ?
Also can you please attach the test results and the terraform configuration file/code..

Thank you